psychopy/tools/versionchooser.py

```diff
@@ -256,7 +255,7 @@ def currentTag():
     """Returns the current tag name from the version repository
     """
     cmd = 'git describe --always --tag'.split()
-    tag = subprocess.check_output(cmd, cwd=VERSIONSDIR).split('-')[0]
+    tag = subprocess.check_output(cmd, cwd=VERSIONSDIR).split(b'-')[0]
```

Comment:
Does this one not need .decode('UTF-8')?

Reply:
Yes, thanks Jon,  I made the change and made another pull request.